### DISCUSSION TITLE
How to configure Hermes SEG for Godaddy

### DISCUSSION THREAD

**Discussion by carlosherz192** [deeztek/Hermes-Secure-Email-Gateway#63] [Support] [answered]
Hello nice project. 
I have a contracted email service with GoDaddy that I would like to configure so that Hermes acts as a gateway between GoDaddy's email service and email clients. I followed the installation steps outlined in the documentation, where the installation was done on a virtual machine to host Hermes SEG, and it is installed on Ubuntu Server 20.04 with 8 GB of RAM and 4 processors.
After the installation, I will configure the network to use a static IP address as follows:

1. Network Mode: Static
2. HostName: smtp
3. Primary Domain Name: domain.tld
4. Ip Address: 192.168.1.50
5. Subnet Mask: 255.255.255.0/24
6. Gateway: 192.168.1.1
7. DNS1: 192.168.1.116

Finally, I added a domain with the following configuration in order for the messages sent and received by email clients to be filtered by Hermes and then forwarded to the GoDaddy server:

1. Domain Name smtp.test.tld
2. Delivery Method: SMTP
3. Recipient Delivery: ANY
4. Destination Address: smtpout.secureserver.net
5. Destination Port: 25
6. Destination Requires Authentication: YES
7. Destination Username: jhon@domain.com
8. Destination Password: ****

Once I have made this configuration, I will try to connect the Outlook application using the following SMTP settings:

1. Smtp Domain: smtp.test.tld
2. Smtp Port: 25

The Outlook email configuration process was completed successfully; however, when I try to send an email, it doesn't go through, and the Hermes SEG log doesn't register anything.
On the other hand, I performed a test by setting up an SMTP client using .NET 8 language with the following configuration:

1. Smtp Domain: smtp.test.tld
2. Smtp Port: 25

In this instance, when I send the email, I get the following log from the Hermes application.

`
PREGREET 22 after 0.01 from [192.168.1.215]:52092: EHLO VMWindows\r\n
`
`
CONNECT from [192.168.1.215]:52092 to [192.168.1.50]:25
`

However, the email is not received by Hermes SEG because it doesn't appear in the Mail Queue view nor does it appear in the Message History view.

So, I would like to know exactly how the configuration should be done in Hermes SEG so that it can act as a gateway between the Godaddy email service and the email clients for the purpose of filtering emails.

What type of client and configuration in the client needs to be done? For example, if the port number is 25 and if the DNS registered in Hermes needs to be specified.
Is it necessary to configure SMTP TLS? If so, could an SSL certificate for a website provided by GoDaddy be used?

Is there a documented use case for configuring Hermes SEG? If so, where can I find this documentation?

**Comment by deeztek** [comment, marked as answer]
Hello,

Hermes SEG can receive external e-mails and relay them to your e-mail server/service. This scenario requires to point your domain's MX record to Hermes SEG. It can also receive e-mails from your e-mail server/service and relay them to their ultimate external destination. This scenario requires for your e-mail server/service to use Hermes SEG as a smarthost/relayhost. In both scenarios Hermes SEG can filter e-mails.

In terms of receiving and/or sending e-mails with your e-mail client, you would configure your e-mail client as normal with your e-mail server/service.

Hopefully that answers your question.

**Comment by deeztek** [comment]
Let us know if you require further assistance.

Thanks
